validators/sessions/session-json-schema.json

```diff
@@ -5,32 +5,45 @@
     "properties": {
         "header": {
             "type": "object",
-            "required": ["typ", "alg"],
+            "required": ["typ", "alg", "kid"],
             "properties": {
                 "typ": {
                     "type": "string"
                 },
                 "alg": {
                     "type": "string"
+                },
+                "kid": {
+                    "type": "string",
+                    "pattern": "^did:ala:(quor|fabr|besu):.+:[0-9a-zA-Z]+#keys-[0-9]+$"
+                },
+                "jwk": {
+                    "type": "string",
+                    "pattern": "^0[xX][0-9a-fA-F]+$"
                 }
             },
             "additionalProperties": false
         },
         "payload": {
             "type": "object",
-            "required": ["@context", "iss", "pku", "data", "iat", "exp"],
+            "required": ["@context", "iss", "type", "alastriaToken", "iat", "exp"],
             "additionalProperties": false,
             "properties": {
                 "@context": {
-                    "type": "string"
+                    "type": "array",
```

Comment:
El JSON schema nos permite indicar aquí un enum donde se indiquen algunos de los valores a cumplir. Por ejemplo puedes usar: 
"@context": {
                            "type": "array",
                            "items": [{
                                "type": "string",
                                "enum": ["https://alastria.github.io/identity/credentials/v1"]
                            }],
                            "additionalItems": true,
                            "uniqueItems": true,
                            "minItems": 1
                        },

Con esto, le dices que como mínimo el array ha de tener 1 ítem, que permite ítems adicionales y un enum para indicar que ese debe de estar. Con esto añadido en el esquema para Context y Type, podéis borrar los tests específicos que lo validan

Reply:
Modificado y borrados los test que sobran por hacer la validación por esquema.
También he actualizado los JWT de ejemplo